src/lib/i18n.ts

```diff
@@ -0,0 +1,82 @@
+import i18next from 'i18next';
+import { createI18nStore } from 'svelte-i18next';
+
+i18next.init({
+	lng: 'en',
+	resources: {
+		en: {
+			translation: {
+				session_one: 'Session',
+				session_other: 'Sessions',
+				knowledge_one: 'Knowledge',
+				knowledge_other: 'Knowledge',
+				settings: 'Settings',
```

Comment:
You don't need `sessions` and `sessionPage`, you can just do:
```
sessions: {
  one: "Session",
  other: "Sessions"
}
```

```
settings: {
  one: "Setting",
}
```

Reply:
Hang on, this is how I usually pluralize things in other projects but that may be different in this library.

Reply:
I'll likely end up moving most of the keys as direct children of `translation: { }`